eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
added new project, because `eclipsePlugin-test` is not built by Travis CI &amp; that is for plugin (scenario) test.